EIPS/eip-2771.md

```diff
@@ -0,0 +1,211 @@
+---
+eip: 2771
+title: Secure Protocol for Native Meta Transactions
+author: Ronan Sandford (@wighawag), Liraz Siri (@lirazsiri),  Dror Tirosh (@drortirosh), Yoav Weiss (@yoavw), Alex Forshtat (@forshtat), Hadrien Croubois (@Amxx), Tomar Sachin (@tomarsachin2271), Patrick McCorry (@stonecoldpat), Nicolas Venturo (@nventuro), Fabian Vogelsteller (@frozeman) 
+discussions-to: https://github.com/ethereum/EIPs/pull/2771
```

Comment:
@MicahZoltu EIP-1 states that pull requests cannot be used as discussion URLs. I'm not here to judge whether that is a good or bad idea, but this draft was merged against EIP-1 rules.

@lightclient is that enforced in eipv?

Reply:
Missed that, good catch!  Definitely something that would be nice for the bot to catch since it is easy to miss (valid Issue URL looks very similar to invalid PR URL).

Reply:
I have listed this as a [requirement](https://github.com/lightclient/eipv#requirements) for `eipv`, but it has not been implemented yet.

Reply:
I issued another PR to redirect the discussion to the following Ethereum Magicians post:

https://ethereum-magicians.org/t/eip-2771-secure-protocol-for-native-meta-transactions/4488